techniques/system/serverRoles/1.0/metadata.xml

```diff
@@ -0,0 +1,39 @@
+
+<TECHNIQUE name="Server roles">
```

Comment:
We should also move the services restart part from rudderApache here since it is a common feature to all server component.